src/providers/sh/commands/deploy.js

```diff
@@ -612,12 +612,12 @@ async function sync({ token, config: { currentTeam, user } }) {
       now.close()
 
       // Show build logs
-      if (!quiet) {
-        if (deploymentType === 'static') {
+      if (deploymentType === 'static') {
+        if (!quiet) {
           console.log(`${chalk.cyan('> Deployment complete!')}`)
-        } else {
-          printLogs(now.host, token, currentTeam, user)
         }
+      } else {
+        printLogs(now.host, token, currentTeam, user)
```

Comment:
This seemed like the best approach to solve the issue, especially since it appears the logger was already designed to operate in quiet mode. But if there's a better way, I'd be happy to update the PR

Reply:
I'd love your thoughts on this @leo @matheuss ? It was said in https://github.com/zeit/now-cli/issues/847 that this issue was a regression, so I'm not sure how it worked before, but this seemed like a solid approach.